modules/claims_api/db/migrate/20190306160555_create_supporting_documents.rb

```diff
@@ -1,13 +1,13 @@
-# frozen_string_literal: true
+# frozen_string_literal: true	
 
-class CreateSupportingDocuments < ActiveRecord::Migration
-  def change
-    create_table :claims_api_supporting_documents, id: :uuid do |t|
-      t.string   :encrypted_file_data, null: false
-      t.string :encrypted_file_data_iv, null: false
-      t.integer :auto_established_claim_id, null: false
-
-      t.timestamps null: false
-    end
-  end
-end
+class CreateSupportingDocuments < ActiveRecord::Migration	
```

Comment:
my linter must've failed me RIP

Reply:
It's alright, not all linters are the same